### PR TITLE
Add integration test support

### PR DESCRIPTION
**Pull request by tjennison-google**

### CODE REVIEW

File: ui/cypress/integration/tests.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe("My First Test", () => {
```
Comment (calbach) [inline ui/cypress/integration/tests.js:1]: nit: update from boilerplate test name

**Review by calbach — APPROVED** [review]
This looks fine to me, but how did you land on Cypress and what were the altneratives considered? I haven't used this before. May be worth writing this up somewhere.

File: ui/cypress/integration/tests.js (hunk 1 above)
Comment (tjennison-google) [reply ui/cypress/integration/tests.js]: Done

**Review by cbookg — APPROVED** [review]

**Comment by cbookg** [comment]
> This looks fine to me, but how did you land on Cypress and what were the altneratives considered? I haven't used this before. May be worth writing this up somewhere.

FYI, the Standards committee recommends Cypress and discusses tradeoffs with other options here:
go/verily-web-ui-integration-testing

**Pull request merged**